topics/authentication.md

```diff
@@ -6,140 +6,151 @@ Code examples: <a href="samples.md" anchor="authentication">Authentication</a>
+* [Basic](basic.md) - an authentication method to provide a user name and password encoded using `Base64`.
+* [Digest](digest.md) - an authentication method that communicates user credentials in an encrypted form by applying a hash function to the username and password.
+* `Bearer` - an authentication scheme that involves security tokens called bearer tokens. You can use [JSON Web Tokens](#jwt) as bearer tokens and use the `jwt` authentication in Ktor to verify a token and validate the claims contained within it.
+
+
+### Form authentication {id="form-auth"}
+[Form](form.md) or form-based authentication uses a [web form](https://developer.mozilla.org/en-US/docs/Learn/Forms) to collect credential information and authenticate a user.
+
+
+### JSON Web Tokens (JWT) {id="jwt"}
+[JSON Web Token](jwt.md) is an open standard for securely transmitting information between parties as a JSON object. You can JSON Web Tokens for authorization: when the user is logged in, each request will include a token, allowing the user to access resources that are permitted with that token. In Ktor, you can verify a token and validate the claims contained within it using the `jwt` authentication.
+
+
+### LDAP {id="ldap"}
+[LDAP](ldap.md) is an open and cross-platform protocol used for directory services authentication. Ktor provides the [ldapAuthenticate](https://api.ktor.io/ktor-features/ktor-auth-ldap/ktor-auth-ldap/io.ktor.auth.ldap/ldap-authenticate.html) function to authenticate user credentials against a specified LDAP server.
+
+### OAuth {id="oauth"}
+[OAuth](oauth.md) is an open standard for securing access to APIs. The `oauth` provider in Ktor allows you to implement authentication using external providers like Google, Facebook, Twitter, and so on.
+
+### Session {id="sessions"}
+[Sessions](sessions.md) provide a mechanism to persist data between different HTTP requests. Typical use cases include storing a logged-in user's ID, the contents of a shopping basket, or keeping user preferences on the client. In Ktor, a user that already has an associated session can be authenticated using the `session` provider. Learn how to do this from [](session-auth.md).
 
-It can be used in some cases along with the [sessions plugin](sessions.md)
-to keep the login information between requests.
 
 ## Add dependencies {id="add_dependencies"}
 To enable authentication, you need to include the `ktor-auth` artifact in the build script:
 
 <var name="artifact_name" value="ktor-auth"/>
 <include src="lib.xml" include-id="add_ktor_artifact"/>
 
-Note that some authentication plugins (for example [JWT/JWK](jwt.md) and [LDAP](ldap.md)) might require additional artifacts.
+Note that some authentication providers, such as [JWT](jwt.md) and [LDAP](ldap.md), require additional artifacts.
+
 
 
-## Basic usage
+## Install Authentication {id="install"}
```

Comment:
This would apply to all instances of this segment:


"This can be the main function ... " -> "This is usually the main function ... "